tests/framework/utils_vhost_user_backend.py

```diff
@@ -9,17 +9,15 @@
 
 from framework import utils
 
-VHOST_USER_SOCKET = "/vub.socket"
 
-
-def spawn_vhost_user_backend(vm, host_mem_path, readonly=False):
+def spawn_vhost_user_backend(vm, host_mem_path, socket_path, readonly=False):
```

Comment:
actually, let's have a default value for `socket_path` in case the calling code doesn't care?

Reply:
This will not be pretty, because we need socket path for `vm.add_vhost_user_block` method.

Reply:
my point is if the caller doesn't care (majority of cases), it will just
```
spawn_vhost_user_backend(vm, host_mem_path)
```
if it does, then
```
spawn_vhost_user_backend(vm, host_mem_path, "socket_path_x")
```

Reply:
So there will be 2 default values: one for `spawn_vhost_user_backend`, another for `add_vhost_user_drive` and they will coincidentally be the same.

Reply:
ok, I see what you mean. Let's keep it as is then.